assets/src/edit-story/components/inspector/prepublish/usePrepublishChecklist.js

```diff
@@ -23,11 +23,29 @@ import { useContext } from 'react';
 import Context from './context';
 
 function usePrepublishChecklist() {
-  const { checklist, refreshChecklist, currentCheckpoint } = useContext(
-    Context
-  );
+  const {
+    checklist,
+    refreshChecklist,
+    currentCheckpoint,
+    focusChecklistTab,
+    isChecklistReviewRequested,
+    resetReviewDialogTrigger,
+    shouldReviewDialogBeSeen,
+    isChecklistEmpty,
+    checklistCount,
+  } = useContext(Context);
 
-  return { checklist, refreshChecklist, currentCheckpoint };
+  return {
+    checklist,
+    focusChecklistTab,
+    refreshChecklist,
+    currentCheckpoint,
+    isChecklistReviewRequested,
+    shouldReviewDialogBeSeen,
+    resetReviewDialogTrigger,
+    isChecklistEmpty,
+    checklistCount,
```

Comment:
oh it's because the checklistCount isn't needed and i missed this reference. 

there's no reason to destructure, it just kinda happened because of the structure early on. I'll fix it here.